common/sagas/config.ts

```diff
@@ -171,12 +168,14 @@ export function* handleNodeChangeIntent(action: ChangeNodeIntentAction): SagaIte
   yield put(setLatestBlock(latestBlock));
   yield put(changeNode(action.payload, actionConfig, actionNetwork));
 
-  const currentWallet: IWallet | null = yield select(getWalletInst);
-
+  // TODO - renable once DeterministicWallet state is fixed to flush properly.
+  // TODO DeterministicWallet keeps path related state we need to flush before we can stop reloading
```

Comment:
Is there a reason we can't implement a `DW_RESET` action, and fire that on network changes?

Reply:
We certainly could, and that's the long-term solution. Probably out of scope here though.